psse/psse-converter/src/main/java/com/powsybl/psse/converter/TransformerConverter.java

```diff
@@ -100,6 +100,8 @@ private void createTwoWindingsTransformer() {
         // As vn2 is used to convert to eu, only the ratio remains to be applied
         TapChanger tapChangerAdjustedYsh = tapChangerAdjustmentAfterMovingShuntAdmittanceBetweenRatioAndTransmissionImpedance(tapChangerAdjustedRatio);
 
+        String name = psseTransformer.getName();
```

Comment:
When the name contains only white chars, it would be better to use `null` (see comments on this PR).